engines/xs/get-latest-version.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the “License”);
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// <https://apache.org/licenses/LICENSE-2.0>.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an “AS IS” BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const get = require('../../shared/get.js');
+
+const getLatestVersion = () => {
+	const url = 'https://github.com/patrick-soquet/moddable-xst/releases';
```

Comment:
Can these downloads be moved to the official Moddable-OpenSource organization? This would better fit jsvu’s policy of using download URLs controlled through the official project’s repository.